script/foreman-config

```diff
@@ -1,48 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
Heads up: this file is mentioned in packaging

Reply:
https://github.com/theforeman/foreman-packaging/pull/4176